src/Repository/Criteria.php

```diff
@@ -230,22 +230,13 @@ public function resolveTime(?string $value, bool $reverse = false): ?string
 
     public function resolveType(?string $value): ?string
     {
-        // @todo
-        $routes = [
-            'all' => 'all',
-            'article' => Entry::ENTRY_TYPE_ARTICLE,
-            'articles' => Entry::ENTRY_TYPE_ARTICLE,
-            'link' => Entry::ENTRY_TYPE_LINK,
-            'links' => Entry::ENTRY_TYPE_LINK,
-            'video' => Entry::ENTRY_TYPE_VIDEO,
-            'videos' => Entry::ENTRY_TYPE_VIDEO,
-            'photo' => Entry::ENTRY_TYPE_IMAGE,
-            'photos' => Entry::ENTRY_TYPE_IMAGE,
-            'image' => Entry::ENTRY_TYPE_IMAGE,
-            'images' => Entry::ENTRY_TYPE_IMAGE,
-        ];
-
-        return $routes[$value] ?? 'all';
+        return match ($value) {
+            'article', 'articles' => Entry::ENTRY_TYPE_ARTICLE,
```

Comment:
same here re: appreciate the refactoring

Reply:
using array map to do lookup is totally fine (and it's quite the oldest trick in the book), but in here I though that using `match` express the intention better, especially when mapping multiple inputs to a single result and also handling defaults (and the fact that we now have `match` at disposal)